resumes/views.py

```diff
@@ -1,3 +1,30 @@
-from django.shortcuts import render
+import boto3
 
-# Create your views here.
+from django.views  import View
+from django.http   import JsonResponse
+from django.conf   import settings
+from django.db     import transaction
+
+from core.utils    import authorization
+from core.storages import S3Client
+from .models       import *
+class ResumeView(View):
+    @authorization
+    def post(self, request):
+        try:
+            with transaction.atomic():
+                resume_file = request.FILES.get('document', None)
+                
+                resume = Resume(
+                    document = S3Client(settings.AWS_ACCESS_KEY_ID, 
+                                        settings.AWS_SECRET_ACCESS_KEY, 
+                                        settings.AWS_STORAGE_BUCKET_NAME).upload(resume_file, 'resume/'),
+                    user  = request.user             
+                )
```

Comment:
성수님 client 객체는 `Resume` 밖에서 만든 뒤 안에서는 `upload`만 해주시면 좋을 것 같네요~
```suggestion
                s3_client = S3Client(...)
                
                resume = Resume(
                    document = s3_client.upload(resume_file, 'resume/'),
                    user     = request.user             
                )
```